DBTEMPLATE/post-refresh/sql/dxh_wrapper_script.sql

```diff
@@ -4,4 +4,6 @@ REM @${SQLDIR}/dxh_reset_user_password.sql
 REM create user testuser1
 REM PROMPT *** executing dxh_custom_create_user.sql ***
 REM @${SQLDIR}/dxh_custom_create_user.sql
+REM PROMPT *** executing dxh_extract_grants_users.sql ***
```

Comment:
These two lines should be dxh_set_grants_users.sql I think

Reply:
Its fixed now